### DISCUSSION TITLE
FILDevSummit Iceland'23: Filecoin Retrievals & Data Availability

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/community#669] [General]
> What's FILDev Summit ➡️ [here](https://github.com/filecoin-project/community/discussions/661).

**Track Leads:** @hannahhoward & @willscott  

Filecoin is a platform for the storage and retrieval of data. While the storage aspect of the protocol has scaled to an exabyte of capacity, this track will focus on retrieval. Retrieval work covers four topics: SLA expectations, measurement of retrieval performance, reputation, and incentivization. Our sessions will build alignment across the community on the ongoing work in these areas.

Full agenda ⏭️ [fildev.io](https://fildev.io/)
Subscribe this discussion to get updates from this track!

**Comment by willscott** [comment]
Calendar for the track is available [here](https://calendar.google.com/calendar/embed?src=c_aa0383d235442553a35de2fe44877d2587d608870f3028344c420478c31442b7%40group.calendar.google.com)
